### ISSUE TITLE
iOS: textarea missing correct contentHeight on content change - event "change" now with correct height (no postlayout needed)

### ISSUE THREAD

**Issue by mbender74** [tidev/titanium_mobile#13413]
### I have searched and made sure there are no existing issues for the issue I am filing

- [X] I have searched the existing issues

### Description

Textarea on iOS missing a correct contentHeight feature.

### Solution

I fixed this on event "change" for textarea which will always return the correct textarea height when the content of textarea is changing.

```
textarea.addEventListener('chage',function(e) {
        console.log(e.textareaHeight);
        console.log(e.value);
});

```

### Platforms

iOS

**Comment by mbender74** [comment]
well I think I will refactor this, so it will go to the "change" event in TiUITextWidgetProxy. m in the noteValueChange() function instead a new event....

**Comment by m1ga** [comment]
was just writing that in the PR 😄 

And if you add `Fixes #13414` in you description it will automatically link this issue with the PR and close it. Also make sure to change the apidoc/ file to reflect the new parameter
